src/GPE.jl

```diff
@@ -102,6 +110,21 @@ end
 
 fit!(gp::GPE, x::VecF64, y::VecF64) = fit!(gp, x', y)
 
+import Base.append!
+append!(gp, x::AbstractArray{Float64, 1}, y::Float64) = append!(gp, reshape(x, :, 1), [y])
+function append!(gp::GPE{X,Y,M,K,P,D}, x::AbstractArray{Float64, 2}, y::AbstractArray{Float64, 1}) where {X,Y,M,K,P <: ElasticPDMat, D}
+    newcov = [cov(gp.kernel, gp.x, x); cov(gp.kernel, x, x) + (exp(2*gp.logNoise) + 1e-5)*I]
+    append!(gp.x, x)
+    append!(gp.cK, newcov)
+    gp.nobs += length(y)
+    append!(gp.y, y)
+    yy = gp.y - mean(gp.mean, gp.x) # would not need to be recomputed every time
+    gp.alpha = gp.cK \ yy
+    gp.mll = -(dot(yy, gp.alpha) + logdet(gp.cK) + log2π * gp.nobs) / 2
```

Comment:
I think it would be good to avoid duplicating code from update_mll!
Either call update_mll! from here, or expect the user (or higher level function) to make a separate call to update_mll! . If the next step is to obtain predictions at other locations, is updating the mll even necessary?